rasgotransforms/rasgotransforms/render/transforms.py

```diff
@@ -0,0 +1,120 @@
+from typing import Callable, Dict, Optional, Union, Tuple
+from pathlib import Path
+import inspect
+import yaml
+
+from rasgotransforms.render.environment import RasgoEnvironment
+from rasgotransforms.main import DataWarehouse
+from rasgotransforms.render.infer_columns import infer_columns
+from rasgotransforms.exceptions import RenderException
+
+__all__ = ["Transforms"]
+
+
+class Transforms(object):
+    """
+    Dynamically adds class methods which render each transform in the package
+    """
+
+    def __init__(self, dw_type: str, run_query: Optional[Callable] = None):
```

Comment:
is there a reason we want to break out `run_query` here vs accepting a dict of `global_overrides` (of which run_query would be one)?

Reply:
Since this environment inherits from a jinja environment, globals can still be passed in here as kwargs. The reason that `run_query` is pulled out is because it is the only macro which does not have a default implementation; every other macro that our transforms use are defined in this environment.